virt/virt-2-6-release-notes.adoc

```diff
@@ -88,6 +89,16 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 
 //CNV-9288: Blank block disks and VMware imported disk images are not preallocated even if requested
 
+* The Containerized Data Importer (CDI) and KubeVirt depend on QEMU which does not support NFS version 3. Therefore, only NFS version 4 is supported. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1892445[*BZ#1892445*])
+
+* The name of the Fedora PVC in the `openshift-virtualization-os-images` namespace is `fedora`, instead of `fedora32`. If you populated the `fedora32` PVC in {VirtProductname} 2.5 or earlier, the virtual machine does not appear in the web console and you cannot use it to clone another virtual machine. To workaround this issue, upload a Fedora image by naming the PVC `fedora`, instead of `fedora32`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1913352[*BZ#1913352*])
+
+* When creating a HPP boot source, the data volume is `pending` with a `WaitForFirstConsumer` status if a user creates the boot source using the `Import via URL (creates PVC)` option. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1929177[*BZ#1929177*])
```

Comment:
This note implies that something is still broken but I see the bug is targeted to 2.6.0 and ON_QA.  Do we need to reword this to say that this issue has been fixed?  Also, I don't believe this issue has ever existed in a released version so why do we need to state that it is fixed?

Reply:
Thanks @aglitke  My bad for not checking the status of the bug. Yes, it makes no sense to document it. 

One question: Would this bug have impacted a z stream release? And if so, does that require we doc it? Just a thought.

But if you agree that a z would have no impact, then I will obsolete the story and update the BZ accordingly.

Reply:
This is still a bug according to QE, though the scope is limited to HPP.